cpp/memilio/math/integrator.h

```diff
@@ -60,11 +60,14 @@ class IntegratorCore
      * @param[out] ytp1 Set to the approximated value of y at time t + dt.
      *     (If adaptive, this time may be smaller, but it is at least t + dt_min, at most t + dt_max.
```

Comment:
Should this be adjusted?

Reply:
I decided against it to not duplicate information, but I think I could just add "If adaptive and force_step_size==false" or something similar to hint at the other parameter.